### PR TITLE
Add tests for Integration Tests

### PR DESCRIPTION
**Pull request by jiahaog**
Taking reference from `integration_ui_driver` and `integration_ui_ios_driver`. The tests were added in https://github.com/flutter/flutter/pull/79258

### CODE REVIEW

**Comment by jiahaog** [comment]
@CaseyHillers can you help to take a look at this please. I'm also not sure if I should change `devicelab_staging_config.star` as well

File: config/devicelab_config.star [hunk 1]
```diff
@@ -457,6 +457,7 @@ def devicelab_prod_config(branch, version, ref):
         "integration_ui_keyboard_resize",
         "integration_ui_screenshot",
         "integration_ui_textfield",
+        "integration_test_test",
```
Comment (CaseyHillers) [inline config/devicelab_config.star:460]: nit: this list is sorted alphabetically

**Review by CaseyHillers — APPROVED** [review]
LGTM!

For staging, we sample prod to test infra changes against. If the integration tests have unique tooling, we should add them to staging.

**Comment by jiahaog** [comment]
Added the configs to staging as well. Would you like to take a look again?

**Comment by CaseyHillers** [comment]
Staging configs LGTM!

**Comment by jiahaog** [comment]
Thanks!

**Pull request merged**